Menu/Menu.js

```diff
@@ -1,10 +1,14 @@
 
 const toggleMenu = () => {
   // Toggle the "menu--open" class on your menu refence. 
+  menu.classList.toggle('menu--open');
 }
 
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu= document.querySelector('.menu');
 // create a reference to the ".menu-button" class
-const menuButton;
+const menuButton= document.querySelector('.menu-button')
 // Using your menuButton reference, add a click handler that calls toggleMenu
+menuButton.addEventListener('click',()=> toggleMenu());
```

Comment:
Because we're not using a class construction no need to use an arrow function here in order to bind the this keyword.
```suggestion
menuButton.addEventListener('click', toggleMenu);
```
When simply referencing a function (not a method0 inside an event listener you can just reference the name as shown above, no need to invoke. The click event will invoke it upon click.